cmd/docker-app/status.go

```diff
@@ -35,6 +37,14 @@ func runStatus(dockerCli command.Cli, claimName string, opts credentialOptions)
 	if err != nil {
 		return err
 	}
+
+	if c.Bundle.Actions == nil {
+		return errors.New(`specified bundle has no "status" action`)
+	}
+	if _, ok := c.Bundle.Actions["status"]; !ok {
+		return errors.New(`specified bundle has no "status" action`)
+	}
```

Comment:
The `RunCustom` which you use below will return `ErrUndefinedAction` if this turns out not to be present, that seems sufficient, not real need to short circuit that here I think.